xapian-applications/omega/templates/styles.css

```diff
@@ -8,19 +8,27 @@ fieldset {
     text-align: center;
 }
 
-fieldset.query-search, .query-metadata,
-.query-suggested-container, .query-options, .query-suggestion {
+fieldset.search, fieldset.suggested-terms,
```

Comment:
Since `fieldset.suggested-terms` has its own block, it's probably easier to copy the couple of rules that aren't overridden down, just for clarity. (It should work fine as is, of course, but there's nothing more annoying than editing CSS and discovering that there's another rule being masked by the current one.)